buildenv/jenkins/JenkinsfileBase

```diff
@@ -678,9 +678,6 @@ def post(output_name) {
 			}
 
 			boolean failIfNoResultsValue = true
-			if (params.PARALLEL && params.PARALLEL != "None") {
-				failIfNoResultsValue = false
-			}
 			step([$class: "TapPublisher", testResults: "**/*.tap", outputTapToConsole: false, failIfNoResults: failIfNoResultsValue])
```

Comment:
Since its only used in line 681, likely do not need to declare a boolean in line 680, just set failIfNoResults: true in line 681.